neonvm-runner/cmd/main.go

```diff
@@ -625,9 +631,10 @@ func newConfig(logger *zap.Logger) *Config {
 		appendKernelCmdline:  "",
 		skipCgroupManagement: false,
 		diskCacheSettings:    "cache=none",
-		memoryProvider:       "", // Require that this is explicitly set. We'll check later.
-		autoMovableRatio:     "", // Require that this is explicitly set IFF memoryProvider is VirtioMem. We'll check later.
-		cpuScalingMode:       "", // Require that this is explicitly set. We'll check later.
+		memoryProvider:       "",             // Require that this is explicitly set. We'll check later.
+		autoMovableRatio:     "",             // Require that this is explicitly set IFF memoryProvider is VirtioMem. We'll check later.
+		cpuScalingMode:       "",             // Require that this is explicitly set. We'll check later.
+		architecture:         runtime.GOARCH, // arm64, amd64
```

Comment:
super tiny nit: can we put this elsewhere in the struct so it doesn't expand the diff?

Reply:
I put it here as all other fields have for the sake of similarity.  Happy to move all three comments to the structure definition though.